docs/autodoc/reader.rst

```diff
@@ -1,51 +1,51 @@
-The Reader API
-==============
+``reader``
```

Comment:
This now looks like code ("`"). As a reader, I would expect to find corresponding code elements (packages or classes). Should't it be the same style as before?

Reply:
Oh yeah, for side by side comparison:
* Master: https://petastorm.readthedocs.io/en/latest/
* New look: https://petastorm.readthedocs.io/en/docs_refactor/

Reply:
In the example you refer to these are the package names, hence it makes sense to use this style. In our case, it's just 'headings'. We should improve our package structure so they correspond, I agree. But for now, IMO it's a little confusing.

Reply:
Ah, that's a valid point.  I may have gone too far in the name of consistency.

Shall I restore the original non-code-block headings?

Reply:
After Slack convo, we concluded that the original heading style is better for now.

So I've restored the heading style and pushed that.